auto/sources

```diff
@@ -333,7 +335,6 @@ HTTP_SRCS="src/http/ngx_http.c \
            src/http/ngx_http_script.c \
            src/http/ngx_http_upstream.c \
            src/http/ngx_http_upstream_round_robin.c \
-           src/http/ngx_http_parse_time.c \
```

Comment:
NOTE: ngx_http_parse_time.c file is not removed in this pr.

Reply:
ack